data/duke.txt

```diff
@@ -0,0 +1,4 @@
+D | 1 | do homework     | 2029-02-20T19:00
```

Comment:
Great format of saving the tasks. A suggestion is that when others fork the repo and install it on local, their tasklists will have these tasks already! But if they were to just use the jar file then I guess it is fine :)